src/numerics/CVodesIntegrator.cpp

```diff
@@ -63,7 +62,7 @@ extern "C" {
         FuncEval* f = (FuncEval*) f_data;
         return f->eval_nothrow(t, NV_DATA_S(y), NV_DATA_S(ydot));
     }
-
+    
```

Comment:
Please don't add any trailing whitespace to lines. You can also configure your editor to remove trailing whitespace.